cmd/swapcli/main.go

```diff
@@ -18,7 +18,6 @@ import (
 )
 
 const (
-	defaultSwapdPort              = 5001
```

Comment:
`swapcli` was defaulting to 5001 and `swapd` was defaulting to 5005. I changed the default for both to 5000.